### PR TITLE
Appcues Events updates [DOC-389]

### LINKED ISSUE #2235: [Content]: Appcues sending events back into Segment
### What article on segment.com/docs is affected?

https://segment.com/docs/connections/destinations/catalog/appcues/

### What part(s) of the article would you like to see updated?

https://segment.com/docs/connections/destinations/catalog/appcues/#sending-appcues-events-to-other-segment-partner-services

In this section it explains how to forward Appcues events to 3rd party tools, which is great, however to help clear up some customer confusion it would be useful to explain how those events will flow back into Segment. So after it says:

> _To enable this feature, go to the Integrations Settings in Appcues and click “Activate” under the Segment integration._

it should go on to say "after you have activated the Segment integration, Appcues will automatically forward these events to your workspace, and will appear in the same source debugger.

It's a very small change, however I've had several customers get very confused so the additional clarity would help save time.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by pwseg**
### Proposed changes

- Added a section on Appcues sending events to Segment, per #2235.
- Removed instances of we/us/ours.
- Clarified lengthy sentences.
- Removed Latin phrases.

### Merge timing

- ASAP once approved

### Related issues (optional)

Closes #2235.

### CODE REVIEW

File: src/connections/destinations/catalog/appcues/index.md [hunk 1]
```diff
@@ -3,39 +3,39 @@ rewrite: true
 title: Appcues Destination
 hide-cmodes: true
 ---
-[Appcues](https://www.appcues.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) adds an experience layer to your product, so you can build user onboarding, NPS surveys, or feature announcements in minutes instead of in weeks. The Appcues JavaScript Destination is open-source. You can browse the code [on GitHub](https://github.com/appcues/analytics.js-integration-appcues).
+[Appcues](https://www.appcues.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) adds an experience layer to your product so you can build user onboarding, NPS surveys, or feature announcements in minutes instead of weeks. The Appcues JavaScript Destination is open-source. You can browse the code [on GitHub](https://github.com/appcues/analytics.js-integration-appcues).
 
-If you notice any gaps, outdated information or simply want to leave some feedback to help us improve our documentation, [let us know](https://segment.com/help/contact)!
+To contribute to this page, visit [Segment’s documentation repository](https://github.com/segmentio/segment-docs/blob/develop/CONTRIBUTING.md).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:8]: Let's remove this as a one-off and explore how we can better add this CTA to all docs.

File: src/connections/destinations/catalog/appcues/index.md [hunk 2]
```diff
@@ -3,39 +3,39 @@ rewrite: true
...
 
-For example, using the server-side destination, customer profile and event data could be directed from a CRM tool into the Appcues platform. This data could then be used for content targeting and user segmentation in the Appcues content editor, alongside data from our `analytics.js` destination.
+As with the JavaScript destination, you can segment and target user profile and event data received through the Appcues server-side Segment destination.
+
+For example, using the server-side destination, you can direct customer profile and event data from a CRM tool into Appcues. You can then use the directed data for content targeting and user segmentation in the Appcues content editor, alongside data from Segment's `analytics.js` destination.
 
 ## Page
 
-If you're not familiar with the Segment Specs, take a look to understand what the [Page method](/docs/connections/spec/page/) does. An example call would look like:
+Refer to the Segment Spec for information about the [Page method](/docs/connections/spec/page/). The following represents an example `page` call:
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:29]: For a long time, the template that our partners used to draft their docs used this "If you're not familiar with..." verbiage. I have a strong dislike for it, and your rewrite is a big improvement.

I'm tempted to do a site wide search / replace, but that's probably risky :)

File: src/connections/destinations/catalog/appcues/index.md [hunk 3]
```diff
@@ -3,39 +3,39 @@ rewrite: true
...
 
 ```javascript
 analytics.page();
 ```
-
-Appcues will check to see if a user qualifies for an experience every time the page changes. When you first call `page` using `analytics.js`, `Appcues.start` checks if there are any current flows for the user and loads them if necessary.
+Each time the page changes, Appcues checks to see if a user qualifies for an experience. When you first call `page` using `analytics.js`, `Appcues.start` checks if there are any current flows for the user and loads them, if necessary.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:34]: I don't think we need `analytics.js` in code format here.

File: src/connections/destinations/catalog/appcues/index.md [hunk 4]
```diff
@@ -44,28 +44,30 @@ analytics.identify('12091906-01011992', {
 });
 ```
 
-When you `identify` on `analytics.js`, we call `Appcues.identify`. This is the preferred method of using and targeting on user properties.
+When you use `identify` on `analytics.js`, Segment call `Appcues.identify`. This is the preferred method of using and targeting on user properties.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:47]: Same as above wrt `analytics.js`.

```suggestion
When you call `identify` with Analytics.js, Segment calls `Appcues.identify`. This is the preferred method of using and targeting on user properties.
```

File: src/connections/destinations/catalog/appcues/index.md [hunk 5]
```diff
@@ -44,28 +44,30 @@ analytics.identify('12091906-01011992', {
...
 
-To get the most out of Appcues, you should send as much user data as possible in the `identify` call. Properties are used to target experiences to specific users and personalize content. Most Appcues customers send properties that fall into a few groups:
-  * Properties to target based on broad classifications such as `role` or `userType`
-  * Properties to personalize Appcues content such as `name`, `firstName` or `company`
-  * Properties to target based on user lifecycle such as `createdAt` (date) or usage metrics such as `numTasksComplete`
+To get the most out of Appcues, you should send as much user data as possible in the `identify` call. Properties personalize content and target experiences to specific users. Most Appcues customers send properties that fall into a few groups:
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:49]:
```suggestion
To get the most out of Appcues, you should send as much user data as possible in the `identify` call. Properties personalize content and target experiences to specific users. Most Appcues customers send properties that fall into one of the following groups:
```

File: src/connections/destinations/catalog/appcues/index.md [hunk 6]
```diff
@@ -81,11 +83,17 @@ Want to read Appcues events in your 3rd party analytics or marketing automation
 
 To enable this feature, go to the Integrations Settings in Appcues and click "Activate" under the Segment integration.
 
+After you activate the Segment integration, Appcues automatically forwards these events to your Segment workspace. The events will appear in the same [Source Debugger](/docs/connections/sources/debugger/).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:86]:
```suggestion
After you activate the Segment integration, Appcues forwards these events to your Segment workspace. The events will appear in the same [Source Debugger](/docs/connections/sources/debugger/).
```

File: src/connections/destinations/catalog/appcues/index.md [hunk 7]
```diff
@@ -81,11 +83,17 @@ Want to read Appcues events in your 3rd party analytics or marketing automation
...
+
 ### Whitelisted Domains
 
-By default, Appcues will target based on the path of the URL. So if we created an Appcues experience and targeted it to `/integrations`, it would appear wherever the embed script is installed with that URL path, like appcues.com/integrations and my.appcues.com/integrations. If your `analytics.js` script is installed on multiple domains (e.g. your marketing site and your web app), you should use Appcues whitelisted domains when targeting your experience.
+By default, Appcues targets based on the URL's path.
+
+For example, if you create an Appcues experience and target it to `/integrations`, it appears wherever the embed script is installed with that URL path, like `appcues.com/integrations` and `my.appcues.com/integrations`.
+
+If you've installed the `analytics.js` script on multiple domains (like a marketing site and your web app), you should use Appcues-whitelisted domains when targeting your experience.
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:94]:
```suggestion
If you've installed the Analytics.js script on multiple domains (like a marketing site and your web app), you should use Appcues-whitelisted domains when targeting your experience.
```

File: src/connections/destinations/catalog/appcues/index.md [hunk 8]
```diff
@@ -81,11 +83,17 @@ Want to read Appcues events in your 3rd party analytics or marketing automation
...
 
 
 ## Delete
 
-When you trigger a user deletion using our [Privacy features](/docs/privacy/user-deletion-and-suppression/), we will forward a delete notification to Appcues, who will act on the notification. You can read more about how Appcues handles deletion requests [in their docs here.](https://docs.appcues.com/article/443-gdpr-deletion-api)
+When you trigger a user deletion using [Segment's Privacy features](/docs/privacy/user-deletion-and-suppression/), Segment forwards an actionable delete notification to Appcues. Refer to the Appcues documentation for information on [how Appcues handles deletion requests](https://docs.appcues.com/article/443-gdpr-deletion-api).
```
Comment (markzegarelli) [inline src/connections/destinations/catalog/appcues/index.md:99]:
```suggestion
When you trigger a user deletion using [Segment's Privacy features](/docs/privacy/user-deletion-and-suppression/), Segment forwards an actionable delete notification to Appcues. Refer to the Appcues documentation for information on [how Appcues handles deletion requests](https://docs.appcues.com/article/443-gdpr-deletion-api){:target="_blank"}.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/appcues/index.md (hunk 2 above)
Comment (pwseg) [reply src/connections/destinations/catalog/appcues/index.md:29]: I didn't even know there was a template!  But that explains why that phrase kept reappearing.

File: src/connections/destinations/catalog/appcues/index.md (hunk 3 above)
Comment (pwseg) [reply src/connections/destinations/catalog/appcues/index.md]: I've reverted it to Analytics.js.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**